src/main/java/org/spongepowered/api/world/extent/Extent.java

```diff
@@ -106,6 +106,25 @@
     }
 
     /**
+     * Get the y value of the highest block that sunlight can reach in the given column.
```

Comment:
Line length is longer than 80 characters. Please run `gradle checkstyle` to fix your warnings.